controller_report_xls/__manifest__.py

```diff
@@ -21,13 +21,13 @@
 ###############################################################################
```

Comment:
Could you run the following command to remove all these old Python headers?

`find . -name "*.py" -type f -exec sed -i '/^[^#]/,$!d; /./,$!d' '{}' \;`

``